text/chipseq_workflow.md

```diff
@@ -0,0 +1,327 @@
+| --------------------------------------------------- | ---------------- | --------------------------- |
+| [ChIP-Atlas] [[1][chip-atlas-1], [2][chip-atlas-2]] | 2018             | Bowtie 2 2.2.2 (2014-04-10) |
+| [ChIPSummitDB] [[1][chipsummitdb-1]]                | 2020             | BWA                         |
+| [Cistrome Project] [[1][cistrome-project-1]]        | 2018             | BWA                         |
+| [Roadmap Epigenomics Project] [[1][rep-1]]          | 2015             | PASH 3.0 (2010)             |
+
+[ChIP-Atlas]: https://chip-atlas.org/
+[chip-atlas-1]: https://doi.org/10.15252/embr.201846255
+[chip-atlas-2]: https://github.com/inutano/chip-atlas/wiki#2-primary-processing
+[ChIPSummitDB]: http://summit.med.unideb.hu/summitdb/
+[chipsummitdb-1]: https://doi.org/10.1093/database/baz141
+[Cistrome Project]: http://cistrome.org/
+[cistrome-project-1]: https://doi.org/10.1093/nar/gky1094
+[Roadmap Epigenomics Project]: http://www.roadmapepigenomics.org/
+[rep-1]: https://doi.org/10.1038/nature14248
+
+Within St. Jude, Computational Biology has historically used BWA to map ChIP-Seq data. Also, the St. Jude Center for Applied Bioinformatics uses BWA as their standard aligner for ChIP-Seq experiments.
+
+## Multiple mapped reads
+
+To reduce false positives, ChIP-Seq experiments commonly filter out reads that are not uniquely mapped.  In order to simplify the ChIP-Seq workflow and to provide the most widely usable alignment data, however, we do not filter reads.
+
+# Specification
+
+With the publication of Brian Abraham's [SEAseq paper], the St. Jude Cloud team has opted to incorporate their ChIP-Seq pipeline as the core alignment method for St. Jude Cloud harmonized ChIP-Seq data. The [SEAseq repository] provides additional details on their method.
+
+[SEAseq paper]: https://doi.org/10.1186/s12859-022-04588-z
+[SEAseq repository]: https://github.com/stjude/seaseq
+
+## Required Metadata
+
+Due to the nature of the experimental methods, ChIP-Seq requires additional metadata to be understood and used for analysis. The following information must be provided for all ChIP-Seq samples to enable their release on St. Jude Cloud.
+
+* Antibody manufacturer/vendor
+* Antibody product/catalog number
+* Antibody lot number
+* Antibody target/mark
+
+## Dependencies
+
+If you'd like the full `conda` environment, you can install it using the following command. Obviously, you'll need to install [anaconda](https://www.anaconda.com/) first.
+
+```bash
+conda create -n chipseq-mapping \
+    -c conda-forge \
+    -c bioconda \
+    picard==2.20.2 \
+    samtools==1.9 \
+    ngsderive==2.2.0 \
+    deeptools==3.5.0 \
+    -y
+```
+
+Additionally, you will want to install our `fqlib` library to check that FastQ files are properly paired and have no duplicate read names. Installation of the [Rust](https://rustup.rs/) programming language is required.
+
+```bash
+cargo install --git https://github.com/stjude/fqlib.git --tag v0.3.1
+```
```

Comment:
1) `fq` is now available via Conda. No need to install rust.
2) These aren't the most recent versions of any of these tools (maybe deeptools is up to date? Didn't check). Where did you get these versions from?
3) I don't see where ngsderive is used in the pipeline. Is there a missing step or is this an unneeded dependency?

Reply:
These are all outdated. This list was based on the original ChIP-Seq workflow that didn't use SEAseq.

Reply:
That's good, with the addition of a link for `SEAseq`. We should definitely avoid documenting SEAseq's dependencies. I'd think just a link to the GitHub should suffice.